core/tests/protractor_utils/ExplorationEditorHistoryTab.js

```diff
@@ -86,7 +86,8 @@ var ExplorationEditorHistoryTab = function() {
         var matched = false;
         for (var i = 0; i < listOfNames.length; i++) {
           if (listOfNames[i] === stateName) {
-            await stateNodes.get(i).click();
+            var getStateNodes = stateNodes.get(i);
```

Comment:
Avoid starting with verbs for variable names. Also, I think you can call this "stateNode" because it represents just one state node.